drivers/misc/nxp_rtxxx_adsp_ctl/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config NXP_RTXXX_ADSP_CTL
+	bool "NXP i.MX RTxxx DSP control"
+	depends on DT_HAS_NXP_RTXXX_ADSP_CTL_ENABLED
```

Comment:
Is the lack of a `default y` here intentional? Usually we add one so all the use needs to do is enable the devicetree node, but I understand if we want to avoid `default y` here since the driver is in the MISC folder (and therefore does not depend on a subystem specific Kconfig)

Reply:
@danieldegrasse It's not actually. It compiles regardless of its inclusion, but I'll add it in.